contributor-dashboard-legacy/src/pages/SizeComparison.tsx

```diff
@@ -262,11 +262,6 @@ const CompareTable = memo(function CompareTable({
 });
 
 function getMainBundleLabel(bundleId: string): string {
-	if (
-		bundleId === "packages/material-ui/build/umd/material-ui.production.min.js"
-	) {
-		return "@mui/material[umd]";
-	}
 	if (bundleId === "@material-ui/core/Textarea") {
 		return "TextareaAutosize";
 	}
```

Comment:
I feel like by default we should use the `bundleId`. `@material-ui/core/Textarea` looks clearer than `TextareaAutosize` to me. I recall Sebastian wanted the `bundleId` to never change so we could always compare sizes with any previous build. While I admire Sebastian for the depth at which he's considering problems, I never felt that this was matching the use case for the tool, so we could remove this constraint. In any cases, if we want to keep it, it still feel like we would have rename rules, but still default to the bundle id.